### ISSUE TITLE
question about loss

### ISSUE THREAD

**Issue by beomgonyu** [yhenon/pytorch-retinanet#225]
in advance, thanks for your code.

1. cls loss.
   I think a value is wrong
   now a value is assigned 0 (back ground) for 0.75, and 1 for 0.25, 
   i think it should be inversed.
2 reg loss
  what is [0.1, 0.1, 0.2, 0.2]] and what for 9.0??
   targets = targets/torch.Tensor([[0.1, 0.1, 0.2, 0.2]])

                regression_loss = torch.where(                    torch.le(regression_diff, 1.0 / 9.0),                    0.5 * 9.0 * torch.pow(regression_diff, 2),                    regression_diff - 0.5 / 9.0                )

thanks

**Comment by R-Sadeghi** [comment]
Thanks for your code.

I have the same question and would be thankful if you answer it.

**Comment by cirorusso2910** [comment]
Hi, I think that:
-  9 is the number of anchor boxes for feature map
- [0.1, 0.1, 0.2, 0.2] is a kind of normalization (mean and standard deviation?) of regression
